docs/provider/provider-tls.md

```diff
@@ -0,0 +1,361 @@
+---
+slug: /provider-tls
+title: TLS Setup
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Provider TLS Setup Guide
+
+:::caution
+All providers on `lava-testnet-2` must use a domain name and TLS. You must have a domain name to continue. If you have not already, please take a moment to purchase one! You can find cheap top-level domains [here](https://www.namecheap.com/) or [here](https://tld-list.com/).
+:::
+
+## 📋 Prerequisites
+
+| Required Setup            |  ?  | 
+| --------------------------|-----|
+| acquired a domain name            | ✅  |
+| `lavad` is installed & configured | ✅  |
+| account with `ulava` balance      | ✅  |
+
+
+## 📝 Written Guide (~45m)
+
+
+<br />
+
+### 🅰️ Change the A Record on your Domain
+
+
+The first step of establishing your Provider is to modify some of the DNS settings on the domain you purchased. In specific, you'll need to change the A Records on your domain. Changing your `A-Record` will create a subdomain that routes traffic to a specific provider process. Depending upon who you've purchased your domain through, A-Records may be visible under `Advanced DNS` or another label.
+
+<details> <summary> 🖧 Multiple Records (Recommended) </summary>
+
+
+We recommend you create a separate `A-Record` for each one of the chains that you plan to support. This is more secure, as the default behavior is to refuse connection unless a consumer connects on the correct subdomain. 
+
+For each chain you want to support, add an `A-Record` with the desired chain name as the `Host`, the `Value` will be your server IP.
+For example, if you wanted to support Ethereum & Lava Mainnets, Your DNS Settings should include the following :
+
+| Record Type       |  Host | Value |
+| ------------------|-------| ----- |
+| A-Record          |   eth |   Your-Server-Public-IP-Address-Here   |
+| A-Record          |  lava |   Your-Server-Public-IP-Address-Here   |
+
+</details>
+
+<details> <summary> ⚀ Single Record </summary>
+
+Alternatively, you can create one `A-Record` that captures traffic to all sub-domains. If you are supporting a large number of chains that frequently changes, doing this may somewhat simplify your process.
+
+| Record Type       | Host | Value |
+|-------------------|------|-------|
+| A-Record          | *    | Your-Server-Public-IP-Address-Here |
+
+</details>
+<br />
+
+### 📂 Update and Install Required Packages 
+
+On the server which you would like to turn into a provider, run updates and install the following packages:
+
+```bash
+sudo apt-get update
+sudo apt install certbot net-tools nginx python3-certbot-nginx -y
```

Comment:
A few comments: `apt update` does not update the system. This command updates the package database. `apt upgrade` is the command to update the system. "run updates" can be misleading in this case, hence my suggestion below. Also, `apt` is a convenient wrapper around `apt-get` for interactive usage.

```suggestion
### 📂 Install Required Dependencies

We will guide you on setting up and configuring Nginx to use a TLS certificate and handle connections to different provider processes. It is also possible to use alternative solutions for these two tasks, such as Caddy, Envoy or your preferred solution.

Run the following commands to install the required packages:

```bash
sudo apt update
sudo apt install certbot net-tools nginx python3-certbot-nginx -y
```

Reply:
Good catch. Adding your change!